### ISSUE TITLE
Coordinator: transaction is not solid: reference transaction is not valid

### ISSUE THREAD

**Issue by viossat** [iotaledger/hornet#470]
I'm running a coordinator (comnet) which after few hours of good operation is now not able to issue its transactions (checkpoints and milestones) at every interval.
Logs: https://ybin.me/p/2a07fefb066a1368#J/v35jXKxC8VUSjaaUPrQhMWCkZW6P4Nl9/E6BuQ3HI=
Config:
```json
{
  "address": "BODHQPXSMDNHBWVZHVATBAHQGZSKWQLXYZNOXMKNUCOZCPTWHHNFBBHFOEGPTWGGUVDJPZAYZIMXIIGVD",
  "securityLevel": 2,
  "merkleTreeDepth": 23,
  "mwm": 10,
  "intervalSeconds": 10,
  "checkpointTransactions": 1
}
```

The warning `transaction is below max depth: reference transaction is not valid` doesn't appear anymore after a restart but `transaction is not solid: reference transaction is not valid` still remains.

**Comment by muXxer** [comment]
`transaction is below max depth: reference transaction is not valid` is because of the fast tangle speed. I guess we have to change that.
For the `transaction is not solid` I have to find a way to synchronize that stuff in a clean way.

**Comment by muXxer** [comment]
Fixed in #476
